finat/tensorfiniteelement.py

```diff
@@ -12,7 +12,7 @@
 
 class TensorFiniteElement(FiniteElementBase):
 
-    def __init__(self, element, shape):
+    def __init__(self, element, shape, transpose=False):
```

Comment:
I think I am more confused by this part because I'm not following which indices are being transposed (and why it appears that not all of the relevant bits need to be)